build/build_apple_frameworks.sh

```diff
@@ -168,7 +168,7 @@ cmake_build() {
         -DEXECUTORCH_BUILD_EXTENSION_DATA_LOADER=ON \
         -DEXECUTORCH_BUILD_EXTENSION_MODULE=ON \
         -DEXECUTORCH_BUILD_EXTENSION_TENSOR=ON \
-        -DEXECUTORCH_BUILD_KERNELS_CUSTOM=$CUSTOM \
+        -DEXECUTORCH_BUILD_EXTENSION_LLM=$CUSTOM \
```

Comment:
If we continue to apply this renaming to the rest of the kernels custom instances, does that mean we're gonna distribute extension llm package eventually on apple platforms together with the executorch runtime?
Please check out what it looks like now for the end users https://pytorch.org/executorch/main/apple-runtime.html
I guess kernels customer are in fact some kernels for llms?

Reply:
@shoumikhin Yeah, that makes sense to me. If we rename them to llm kernels, do I just need to update the frameworks build as well?